Sources/Stagehand/Animation/Animation.swift

```diff
@@ -62,23 +62,23 @@ public struct Animation<ElementType: AnyObject> {
     /// The duration of the animation.
     ///
     /// More specifically, this is the duration of one cycle of the animation. An animation that repeats will take a
-    /// total duration equal to the duration of one cycle (the animation's `duration`) multiplied by the number of
-    /// cycles (as specified by the animation's `repeatStyle`).
+    /// total duration equal to the duration of one cycle (the animation's `implicitDuration`) multiplied by the number
+    /// of cycles (as specified by the animation's `implicitRepeatStyle`).
```

Comment:
I want to follow this up with an effort to be more consistent with the terminology we use in the project - both in the long form documentation and headerdocs, as well as some of the (private) variable names. For example, we talk about the "total" duration here, but I don't think that's the most descriptive term. When talking about durations, I've started differentiating by saying:

* The time from the start of the execution phase to when the animation completes is the `end-to-end duration`. This is usually a calculated value based on the cycle duration and the repeat style (when available).
* The time for the animation to go from a relative timestamp of 0 to 1 (or 1 to 0) is the `cycle duration`. This is typically how consumers specify the duration.
* The time for the animation to go between two arbitrary relative timestamps is the `segment duration`. This is used in a few places like snapshot testing and the upcoming interactive animations.

Reply:
This makes sense, `total` can be ambiguous when animation has multiple phases. `end-to-end` sounds good but feels a bit clumsy as variable name, what about `overall duration`?

Reply:
Yeah, I could see "overall" working for that. I filed to #52 to track updating the documentation.